bumble/host.py

```diff
@@ -583,31 +595,18 @@ def send_l2cap_pdu(self, connection_handle: int, cid: int, pdu: bytes) -> None:
             offset += data_total_length
             bytes_remaining -= data_total_length
 
-    def supports_command(self, command):
-        # Find the support flag position for this command
-        for octet, flags in enumerate(hci.HCI_SUPPORTED_COMMANDS_FLAGS):
-            for flag_position, value in enumerate(flags):
-                if value == command:
-                    # Check if the flag is set
-                    if octet < len(self.local_supported_commands) and flag_position < 8:
-                        return (
-                            self.local_supported_commands[octet] & (1 << flag_position)
-                        ) != 0
-
-        return False
+    def supports_command(self, op_code: int) -> bool:
+        return (
```

Comment:
I recommend using `HCI_SUPPORTED_COMMANDS_MASKS.get()` instead of `HCI_SUPPORTED_COMMANDS_MASKS[]` to avoid an exception if the `op_code` isn't in the map (ex: a caller asking about an unknown/new/vendor op_code).

Reply:
Done